requirements.txt

```diff
@@ -1,23 +1,3 @@
-Adafruit-GPIO==1.0.1
```

Comment:
I think you're cutting too much out. I'd rather have a big requirements file than have a broken one.